`Synapse`: enable user assigned managed identity for Synapse workspace (#16429)
hashicorp/terraform-provider-azurerm: website/docs/r/synapse_workspace.html.markdown

---

> An `identity` block supports the following:

Reviewer: in this case we can only have one

```suggestion
The `identity` block supports the following:
```

> * `type` - (Required) Specifies the type of Managed Service Identity that should be configured on this Synapse Workspace. Possible values are `SystemAssigned`, `UserAssigned`, `SystemAssigned, UserAssigned` (to enable both).

Reviewer: ```suggestion
* `type` - (Required) Specifies the type of Managed Service Identity that should be associated with this Synapse Workspace. Possible values are `SystemAssigned`, `UserAssigned` and `SystemAssigned, UserAssigned` (to enable both).
```
